normandy/recipes/tests/test_models.py

```diff
@@ -103,6 +120,31 @@ def test_canonical_json(self):
         expected = expected.encode()
         assert recipe.canonical_json() == expected
 
+    def test_signature_is_invalidated(self):
+        recipe = RecipeFactory(name='unchanged', signed=True)
+        recipe.name = 'changed'
+        recipe.save()
+        recipe = Recipe.objects.get(id=recipe.id)
```

Comment:
You can use [refresh_from_db](https://docs.djangoproject.com/en/1.9/ref/models/instances/#django.db.models.Model.refresh_from_db) here and below.